content/docs/1.7.0/deploy/install/_index.md

```diff
@@ -106,6 +106,11 @@ Use the `install` sub-command to install and set up the preflight dependencies b
 
 ### Using the Environment Check Script
 
+```suggestion
+  Since Longhorn v1.7.0, we have introduced the [Longhorn Command-line Tool](../../advanced-resources/longhornctl/). The functionality of the environment check script, [environment_check.sh](https://github.com/longhorn/longhorn/blob/master/scripts/environment_check.sh) overlaps
+with that of the Longhorn Command-line Tool. Therefore the script `environment_check.sh` has been deprecated in v1.7.0 and is scheduled for removal in v1.8.0.
+```
+
```

Comment:
Also need to use `Notice` quote

```suggestion
  Since Longhorn v1.7.0, we have introduced the [Longhorn Command-line Tool](../../advanced-resources/longhornctl/). The functionality of the environment check script, [environment_check.sh](https://github.com/longhorn/longhorn/blob/master/scripts/environment_check.sh) overlaps
with that of the Longhorn Command-line Tool. Therefore, the script has been deprecated in v1.7.0 and is scheduled for removal in v1.8.0.
```

```

Reply:
Modified. Thanks.